README.md

```diff
@@ -238,6 +238,15 @@ Files can be added to an instance with `addFiles`.
 $instance->addFiles($fileOrDirectoryOnHost, $pathInContainer);
 ```
 
+#### Get the docker inspect information
+
+The json decoded array from the docker inspect command can be retrieved with `inspect`.
+
+```php
+$inspectArr = $instance->inspect();
```

Comment:
Please don't use abbreviations. 

Could you also add some example output here?

Reply:
Just added the examples.